README.rst

```diff
@@ -21,6 +21,14 @@ Features
 * Works with the `Docker for Mac <https://docs.docker.com/docker-for-mac/>`_ and `Docker for Windows <https://docs.docker.com/docker-for-windows/>`_.
 
 
+What is the difference between `dockcross` and `dockbuild` ?
+------------------------------------------------------------
+
+The key difference is that `dockbuild <https://github.com/dockbuild/dockbuild>`_
+images are using the convenient approach of `dockcross` but they do **NOT**
```

Comment:
images are using the convenient approach of -> images use the same method conveniently isolate the build environment as